figures/3.figure3/scripts/2.add_scale_bars.py

```diff
@@ -54,21 +54,21 @@
     "image_Mitochondria_crop_path",
 ]
 for path_type in image_path_columns:
-    print(path_type)
+    # print(path_type)
     for image_path in df[path_type]:
-        print(image_path)
         # open the image
         image = cv2.imread(image_path, cv2.IMREAD_UNCHANGED)
         # image to array
         image_array = np.array(image)
         resolution = 3.3446  # pixels per micrometer
         # get the image size
         image_size = image_array.shape
-        if image_array.shape[1] <= 100:
+        print(image_size)
```

Comment:
i recommend removing these testing-associated lines prior to merging. If they are not testing-associated, then add a comment about why it is necessary (since the print statement itself is not obvious during runtime what it would be)